case-lib/lib.sh

```diff
@@ -571,6 +575,34 @@ func_lib_check_sudo()
     }
 }
 
+func_lib_enable_pipewire()
+{
+    dlogi "Starting Pipewire..."
+    
+    sudo systemctl --global unmask pipewire{,-pulse}.{socket,service}
+    systemctl --user unmask pipewire{,-pulse}.{socket,service}
+    systemctl --user start pipewire{,-pulse}.{socket,service}
+    sudo systemctl --global unmask wireplumber.service 
+    systemctl --user unmask wireplumber.service
+    systemctl --user start wireplumber.service
```

Comment:
I would unmask everything first to get back to the normal OS state, and then start.